code/game/objects/items/airbubble.dm

```diff
@@ -357,6 +357,9 @@
 			return
 		user.drop_item()
 	else if(istype(W, /obj/item/weapon/handcuffs/cable))
+		if(zipped)
+			user.visible_message("<span class='warning'>[src]'s zipper is already cuffed.</span>")
```

Comment:
`retrained` seems like more appropriate word in this context than `cuffed`

Reply:
Changed